### DISCUSSION TITLE
How can I help?

### DISCUSSION THREAD

**Discussion by tim-gromeyer** [YimMenu/HorseMenu#16] [Q&A] [answered]
Hi there,

I would like to help with the development, but unfortunately I don't have any experience with DirectX12 or Vulkan. Also, I'm using Linux, so I'm having problems compiling the menu.

Is there any way I can help with development even with the limited knowledge/skills I have?

**Comment by MarkEcza** [comment]
Creating issue's and making suggestions.

**Reply by tim-gromeyer** [reply]
I'll do that, thanks!

**Comment by Yimura** [comment, marked as answer]
The easiest way of developing on Linux is to setup a Windows VM, install an SSH server on there and then from Linux remote into the VM to develop and compile the mod menu (using Visual Studio Code).

As for development, gain experience yourself by experimenting. The current base is very much in its infancy so there isn't much to do for people that aren't a little experienced with mod menu development.

**Reply by tim-gromeyer** [reply]
> The easiest way of developing on Linux is to setup a Windows VM, install an SSH server on there and then from Linux remote into the VM to develop and compile the mod menu (using Visual Studio Code).

I've tried it, but it's not easy. I couldn't get MSVC to work, it kept failing with `cannot open kernel.lib`. After a few hours I gave up and deleted everything (Visual Studio (Code), Windows SDK, some Qt tools)

> As for development, gain experience yourself by experimenting. The current base is very much in its infancy so there isn't much to do for people that aren't a little experienced with mod menu development.

This is what I would have done, but Windows doesn't work. It is not designed for terminal use and is completely bloated. Visual Studio, the only way I found to install MSVC, is also bloated and takes up a lot of space for a bloated IDE I don't want. I just want MSVC and use it with Qt Creator. Can you help me with this? Just get MSVC to work without installing Visual Studio?

**Comment by Tayipc38** [comment]
If you mean to contribute in terms of features, I'd suggest you wait until the base has its essential assets in place. There isn't even an UI in place yet.
